repositories/database_methods.py

```diff
@@ -209,4 +209,49 @@ def edit_post(post_id: int, post_content: str) -> bool:
                             WHERE
                                 post_id = %s
                             ''', [post_content, post_id])
-            # what can I return here?
\ No newline at end of file
+            # what can I return here?
+
+def get_user_concentration(user_id: int) -> dict[str, Any] | None:
+    pool = get_pool()
+    with pool.connection() as connection:
+        with connection.cursor(row_factory=dict_row) as cursor:
+            cursor.execute('''
+                            SELECT 
+                                concentration
+                            FROM 
+                                users
+                            WHERE 
+                                user_id = %s
+                            ''', [user_id])
+            concentration = cursor.fetchone()
+            return concentration
```

Comment:
You can probably just add this into "get_user_by_id" as well so that you don't have to use both methods at the same time just to get the user's information.